src/components/collapsable/collapsable.css

```diff
@@ -0,0 +1,46 @@
+oui-collapsable .oui-collapsable__title {
+  display: flex;
+  align-items: center;
+  border: 1px solid rgba(0, 0, 0, 0.3);
+  border-radius: 0.25em 0.25em 0 0;
+  padding: 0.5em 0.3em 0.5em 0;
+  user-select: none;
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
+  cursor: pointer;
+}
+
+oui-collapsable .oui-collapsable__title:hover {
+  border-color: rgba(0, 0, 0, 0.5);
+}
+
+oui-collapsable .oui-collapsable__title oui-svg {
+  margin: 0 0.6em;
+  transform: rotate(180deg);
```

Comment:
The common pattern with expandable sections (similar to github's source expansion) is to represent "closed" state, along with standard summary / details elements, with an arrow pointing at the title of the section